draft-ietf-quic-transport.md

```diff
@@ -5730,6 +5730,17 @@ decisions are made independently of client-selected values; a Source Connection
 ID can be selected to route later packets to the same server.
 
 
+## Hash Collision Attack on Stateless Reset Tokens
+
+As the stateless reset tokens are controlled by the peers, QUIC endpoints SHOULD
+be resistant to hash flooding DoS attacks when using a hash map for retaining
+those tokens. One way of achieving such property is to retain and compare the
+transformed values of the stateless tokens where the transformation is defined
+as a pseudo-random permutation (e.g., block cipher) or a keyed hash (e.g., HMAC
+{{?RFC2104}}) that is cryptographically secure, instead of using the raw token
+values as the hash keys.  This approach satisfies the secure comparison
```

Comment:
Suggested rephrase: "... transformed values of the stateless tokens instead of the raw token values. A transformation such as a pseudo-random permutation (e.g. block cipher) or a cryptographically-secure keyed hash (e.g., HMAC {{?RFC2104}}) can be used to also ensure that comparing stateless tokens does not leak timing information, as specified in {{detect-stateless-reset}}."